tests/runtest.sh

```diff
@@ -370,6 +370,8 @@ function create_core_overlay {
         rm -f "$coreOverlayDir/System.Private.CoreLib.ni.dll"
     fi
     copy_test_native_bin_to_test_root
+
+    chmod +x $coreOverlayDir/corerun
```

Comment:
This is due to an obscure problem with jenkins archiving on arm64. The executable bit is not preserved when we archive in jenkins. If you would prefer for this to be set outside of runtests in groovy for arm64 before running runtests that is possible as well.